packages/paste-core/components/form/src/FormLabel.tsx

```diff
@@ -35,6 +35,7 @@ const FormLabel: React.FC<FormLabelProps> = ({as, marginBottom, required, disabl
     <Box
       {...safelySpreadBoxProps(props)}
       as={as}
+      borderBottomWidth="borderWidth0"
```

Comment:
Is it possible to leave a comment here to warn our future selves why we're setting this?